### PR TITLE
Correct release notes entry in 377

### PR DESCRIPTION
**Pull request by mosabua**
## Description

Correct a release notes entry for Trino 377 for the Hive connector where @martint and myself misread the comments and linked tickets.

> Is this change a fix, improvement, new feature, refactoring, or other?

Fix. 

> Is this a change to the core query engine, a connector, client library, or the SPI interfaces? (be specific)

Docs

> How would you describe this change to a non-technical end user or system administrator?

Fix a minor detail in the release notes.

## Related issues, pull requests, and links

na.

## Documentation

( ) No documentation is needed.
(x) Sufficient documentation is included in this PR.
( ) Documentation PR is available with #prnumber.
( ) Documentation issue #issuenumber is filed, and can be handled later.

## Release notes

(x) No release notes entries required.
( ) Release notes entries required with the following suggested text:

### CODE REVIEW

File: docs/src/main/sphinx/release/release-377.md [hunk 1]
```diff
@@ -7,9 +7,10 @@
 
 ## Hive connector
 
-* Add support for partition names that include special characters. ({issue}`11719`)
```
Comment (mosabua) [inline docs/src/main/sphinx/release/release-377.md:10]: This linked was linked from the one we now add and we thought this is fixed.

File: docs/src/main/sphinx/release/release-377.md [hunk 2]
```diff
@@ -7,9 +7,10 @@
...
 * Add support for `date` type partition names with timestamp formatting. ({issue}`11873`)
 * Improve performance of queries that use Glue metadata. ({issue}`11869`)
+* Fix `sync_partition_metadata` procedure when partition names differ from
```
Comment (findinpath) [inline docs/src/main/sphinx/release/release-377.md:12]: `sync_partition_metadata` can cope with partition names which differ from the partition paths on the file system

**Review by findinpath — APPROVED** [review]

File: docs/src/main/sphinx/release/release-377.md (hunk 2 above)
Comment (mosabua) [reply docs/src/main/sphinx/release/release-377.md:12]: Thats not a full sentence .. also still missing what this did before

E.g. 

Prevent failures of ...

Comment (mosabua) [reply docs/src/main/sphinx/release/release-377.md:12]: Well... still not sure what happened before.. did data get corrupted or did the procedure just fail or something else?

Comment (findinpath) [reply docs/src/main/sphinx/release/release-377.md:12]: Due to the fact that the comparison mechanism between the partition names stored in HMS and partition paths on the file system/ object storage was incorrect, when trying to sync the partition metadata the second time for a table, the user kept hitting rather puzzling exceptions. 

When using 'FULL' mode after the first successful 'FULL' mode run:

```
dropping a partition added in the same transaction is not supported: testschema.test_table_external [2021-04-25, 20:00:13, 12345]
```
When using 'ADD' mode after the first successful 'FULL' mode run :

```
One or more partitions already exist for table 'testschema.test_table_external'
```

This was due to the fact that partition paths corresponding to existing partition names in HMS were falsely marked to be added/removed from HMS, reason why HMS threw the exceptions mentioned earlier. 

I hope this comment gives a better context of the problem.

Comment (mosabua) [reply docs/src/main/sphinx/release/release-377.md:12]: Okay .. thats rather tricky. Do you have any suggestions on how to make a useful release notes entry out of this @martint ?

Maybe just 

Prevent failures of the `sync_partition_metadata` system procedure for inconsistent partition names stored in Hive metastore and used in the file system.

**Comment by mosabua** [comment]
@findepi why was this merged .. i thought we aim to fix the message to clarify as discussed in the thread .. should I send another follow up @martint ?

**Comment by findepi** [comment]
i thought there is an agreement already, sorry

**Pull request merged**